### ISSUE TITLE
support picture in picture for videos

### ISSUE THREAD

**Issue by DetachHead** [moffatman/chan#196]
it would be nice if you could watch videos while using other apps, like you can with the youtube app

**Comment by moffatman** [comment]
It would be nice yeah. Rather massive amount of work though, don't expect it any time soon.
By the way are you on iOS or Android?

**Comment by DetachHead** [comment]
ios

**Comment by moffatman** [comment]
Yes that's the difference, would have to use Android native picture-in-picture API. Vs. IOS
